eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/comm/Job.java

```diff
@@ -160,14 +168,19 @@ public void setLinks(List<Link> links) {
 		return messages;
 	}
 
-	public JobListRow toJobListRow() {
+	public JobListRow toJobListRow(boolean getStatisticsSupported) {
```

Comment:
Having to pass in getStatisticsSupported seems redundant because there is a getStatisticsSupported field.